spec/features/user/log_mood_spec.rb

```diff
@@ -54,5 +64,6 @@
     expect(page).to have_content 'Welcome, ' + user.name
     expect(mood_entry.user_id).to eq user.id
     expect(mood_entry.mood_id).to eq @mood1.id
+    expect(mood_entry.entry).to eq "I'm feeling sad actually"
   end
 end
```

Comment:
I forgot to bring it up in retro but did we make the diary entry available to view anywhere or is that something to add later?